src/routes/api.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+import express from 'express';
+import modelFinder from '../middleware/modelFinder';
+import auth from '../auth/middleware';
+import Team from '../models/team';
+import User from '../auth/user';
+const router = express.Router();
+
+router.param('model', modelFinder);
+
+/***********************************
+*     POST REQUESTS                *
+************************************/
+router.post('/team', auth, (req, res, next) => {
+
+  if (req.user.role === 'coach') {
+    req.body.coach = req.user._id;
+
+    let document = new Team(req.body);
```

Comment:
Here you know for a fact it is a Team object, so calling it 'team' instead of 'document' would be recommended. Whereas in your other :model routes it could be any type of model, so using a more general variable name like 'document' would be ok.